solutions/swb-ui/scripts/updatePlugins.sh

```diff
@@ -56,5 +56,5 @@ for page in ../swb-ui/src/pages/apps/*; do #Iterate through all items in pages/a
 done 
 echo "Running rush update";
 rush update;
-echo "Fininshed Updating of Plugins";
+echo "Finished Updating of Plugins";
```

Comment:
Also make sure that package.json of swb-ui has Create_React_App removed

Reply:
I'm sorry, what's the action item for the comment above? Is it additional text I should add to the message?

Reply:
Sorry ignore the item. I thought that the package.json of swb-ui wasn't updated with the removed dependency, but it's correct.